### PR TITLE
Correct support for destructuring assignments in symbol table and CFG

### LINKED ISSUE #179: Fix wrong order of array and object destructuring assignments in CFG
This :
```
a = 1;
b = 2;
c = 3;
[a, b, c] = doSmth([a, b, c]);
```
results in a Block with the left side of the assignment happening before the right side of the assignment. This impacts the Live Variable Analysis results

### PR DESCRIPTION
**Pull request by inverno**
Fixes #179

### CODE REVIEW

File: src/symbols/lva.ts [hunk 1]
```diff
@@ -55,36 +55,35 @@ export class LiveVariableAnalyzer {
...
-        }
-        if (usage.is(UsageFlag.READ)) {
-          availableReads.add(usage.symbol);
-        }
+      if (isAssignment(element)) {
+        siftIdentifiers((element as ts.BinaryExpression).left).identifiers.forEach(identifier => {
+          this.trackUsage(this.symbols.getUsage(identifier), availableReads);
+        });
+      } else {
+        this.trackUsage(this.symbols.getUsage(element), availableReads);
```
Comment (vilchik-elena) [inline src/symbols/lva.ts]: @inverno why? We didn't have any logic before.

**Comment by vilchik-elena** [comment]
@inverno So finally do we process every kind of element in object destructuring assignment?

File: src/symbols/lva.ts (hunk 1 above)
Comment (inverno) [inline src/symbols/lva.ts]: @vilchik-elena in fact we did, it was inside the `usageNode` method. Look at the code as it was a few commits before, like in f943a7f0f311399dc28b12a943d0b56051225a76

**Comment by inverno** [comment]
@vilchik-elena I hope! Didn't check it

**Pull request merged**